### ISSUE TITLE
Cluster doesn't work if the network service starts after the cluster

### ISSUE THREAD

**Issue by jesuslinares** [wazuh/wazuh#1355]
After a restart, worker node was not able to connect, sending retries every ten seconds but never able to connect until the master node was restarted.

It looks like the Manager daemons were starting even **before network service** was ready, causing an "Unreachable network" error in Wazuh cluster and corrupting the master node connections table. 

It could be related to https://github.com/wazuh/wazuh/issues/673.

**Comment by jesuslinares** [comment]
## Simulate error

This error can be simulated following these steps:
- Connect master and worker.
- Unattach network from worker.
- Re-start worker node.

The worker node is closed without notifying the master (there is no network), so the master considers the worker as connected, rejecting the worker node since "it is connected".

# Solution

- The workers nodes are sending a keep-alive every minute.
- The master node should remove a worker if a keep-alive is not sent in the last 2 minutes (+ 10s delay).

**Comment by mgmacias95** [comment]
Hi @jesuslinares,

The implemented solution is to simply break the connection between master and worker if no keepalives can be sent or received. The connection must be broken from both sides.

In the master side, a new thread has been added. This thread checks every minute the last keep alive received from each worker. A worker is disconnected if no keepalive has been received in the last 2 min:
https://github.com/wazuh/wazuh/blob/4c7202473411d28fc3d8476db28d82a26e1cde2f/framework/wazuh/cluster/master.py#L717-L724

In the worker side, the number of fail attempts to send a keepalive is stored. When this number is high enough, the client is disconnected.
https://github.com/wazuh/wazuh/blob/4c7202473411d28fc3d8476db28d82a26e1cde2f/framework/wazuh/cluster/worker.py#L604-L617
After some testing, I found out that it took 20min to fully break the connection and try to connect again:
```
2018-09-28 12:58:24,787 INFO    : [Worker] [AgentInfo-S  ]: End. Sleeping: 60s.
2018-09-28 13:13:04,757 INFO    : [Transport-ClientHandler] Disconnected.
```

Best regards,
Marta
